recipes/python-benedict/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "python-benedict" %}
+{% set version = "0.22.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python-benedict-{{ version }}.tar.gz
+  sha256: 2a5fda1307b0f0ecaf308706a169e31c7141bfbe03df8775d72be3b93d09091d
+
+build:
+  number: 0
```

Comment:
Any reason not to add `noarch:python` here?

Reply:
I think `grayskull` left out the `noarch:python` because benedict's `setup.py` defines a pinned version for `ftfy` on python 2.7.

I've fixed the `ftfy` entries in requirements/run, and my understanding is that we don't want `noarch:python` here.

One thing I wasn't sure of is whether or not `skip: true  # [py<27]` is actually needed?

Reply:
No, as is this recipe can't be noarch python. However, by default conda-forge is not building Python 2.7 packages--do you have need for the Python 2.7 support?

Reply:
Not particularly, `noarch: python` and `python >=3.4` would work fine for us if that is preferred at this point.

Reply:
(or whatever the default min python3 version is for conda-forge now)

Reply:
The big thing is that with noarch, only one package needs to be built. Without it, you need a package for every supported combination of architecture and Python version including PyPy (currently over a dozen).

Reply:
So yes, if you could make it noarch and `python >=3.4` (or whatever works for you--we're only building new packages for >=3.6, but it doesn't matter much for noarch so long as the code runs) that would be greatly preferable.